modules/install-sno-generating-the-install-iso-manually.adoc

```diff
@@ -116,7 +116,7 @@ sshKey: |
 <4> Set the `metadata` name to the cluster name.
 <5> Set the `networking` details. OVN-Kubernetes is the only allowed networking type for single-node clusters.
 <6> Set the `cidr` value to match the subnet of the {sno} cluster.
-<7> Set the path to the installation disk drive, for example, `/dev/sda`.
+<7> Set the path to the installation disk drive, for example, `/dev/disk/by-id/wwn-0x64cd98f04fde100024684cf3034da5c2`.
```

Comment:
Suggestion:
You could add `<disk_id>` in the yaml since the user may not know what the id in example is.

Reply:
Addressed